src/Rule/ForbidUselessNullableReturnRule.php

```diff
@@ -80,4 +85,9 @@ public function processNode(
         return [];
     }
 
+    private function isOverridable(MethodReflection $methodReflection): bool
+    {
+        return !$methodReflection->isFinal()->yes() && !$methodReflection->isPrivate() && !$methodReflection->isStatic() && !$methodReflection->getDeclaringClass()->isFinal();
```

Comment:
`$methodReflection->getDeclaringClass()->isFinalByKeyword()` is needed as `/** @final */` classes are still actually overridable